interface/orders/orders_results.php

```diff
@@ -163,7 +163,7 @@ function QuotedOrNull($fld)
 <link rel="stylesheet" href='<?php  echo $css_header ?>' type='text/css'>
 <link rel="stylesheet" href="<?php echo $GLOBALS['assets_static_relative']; ?>/jquery-datetimepicker/build/jquery.datetimepicker.min.css">
 
-<title><?php  xl('Procedure Results', 'e'); ?></title>
```

Comment:
In these older script, you will note that the xl function has a second variable with `'e'`, which basically does an echo on the translation so did not need to put an `echo` in front of it.
We decided to deprecate that feature, so in these cases need to remove the e parameter and add an echo to the front if it like so:
```
<title><?php echo xlt('Procedure Results'); ?></title>
```

Reply:
I'll go ahead and correct these instances - thank you!!